sys/mips/exc.S

```diff
@@ -21,36 +21,48 @@
         .local skip_fpu_save
         .local skip_fpu_restore
 
+        .cfi_sections .debug_frame, .eh_frame
+
 # [$k0] must be set to value of C0_STATUS
 exc_enter:
         mfc0    k0, C0_STATUS           
         andi    k0, SR_KSU_MASK         # Did exception occur in kernel mode?
         beqz    k0, kern_exc_enter
         nop
 
+
 # IMPORTANT: While exception level is active the code may only access kernel
 # stack and local pcpu structure! FTTB These are accessed through kseg0.
 # It is UNSAFE to use k0/k1 registers when TLB miss handler may be triggered!
 
 NESTED(user_exc_enter, USER_EXC_FRAME_SIZE, ra)
+        .cfi_signal_frame
```

Comment:
This one is really required. Without it backtrace stops at first stack frame in user space. Good job!